### ISSUE TITLE
Outdated subscription events

### ISSUE THREAD

**Issue by cBrenDev** [ZaiusInc/zaius-magento-2#4]
https://github.com/ZaiusInc/zaius-magento-2/blob/0a0dbc931f443a655bfdfe06859c75e29267e321/Observer/NewsletterSubscriberSaveObserver.php#L47-63

We've migrated to using type list, action subscribe, and a list_id (which is meant to be configurable on a store view level, with the default list_id being "newsletter"). Looks like we missed upgrading M2 when we got to this.

Both of these can be events. The first (addEventToSession) should use the configured list_id, or "newsletter" by default. The second (postEntity) should be changed to be a matching event, but would use the static list_id "zaius_all" instead.

Finally, the events should send vuid & vtsrc like all other backend events.

**Comment by travish** [comment]
This is fixed, but not up yet. We need to discuss what's happening here:

https://github.com/ZaiusInc/zaius-magento-2/blob/0a0dbc931f443a655bfdfe06859c75e29267e321/Observer/NewsletterSubscriberSaveObserver.php#L46

This line checks for admin area (whether the user is in adminhtml), so it will effectively send *both* now on frontend subscription events.

Other than that, here's the changes:

Both are postEvents now.

Both are equal in their payloads (except the static list_id behavior):

```
type = list
action = subscribe
list_id = list_id
subscribe = true|false
vuid = vuid
vtsrc = vtsrc
```

Added the field for Global ID Prefix to the Settings Configuration.
Added the field for Newsletter ID to the Settings Configuration.

**Comment by cBrenDev** [comment]
Fixes went live in 1.0.0-beta.3
